examples/samples.h

```diff
@@ -13,4 +13,9 @@ int canon_enable_capture (Camera *camera, int onoff, GPContext *context);
 extern int camera_auto_focus (Camera *list, GPContext *context, int onoff);
 extern int camera_eosviewfinder (Camera *list, GPContext *context, int onoff);
 extern int camera_manual_focus (Camera *list, int tgt, GPContext *context);
+
+#if !defined (O_BINARY)
```

Comment:
is O_BINARY really needed to save files in binary mode? or would this just be for fopen?